SCClassLibrary/Common/Collections/Dictionary.sc

```diff
@@ -296,12 +296,17 @@ Dictionary : Set {
 
 	isAssociationArray { ^false }
 
-	asPairs {
-		^this.getPairs
+	asPairs { |class|
+		var res = (class ? Array).new(this.size * 2);
+		this.pairsDo { |key, val|
+			res.add(key).add(val);
+		};
+		^res
 	}
 
-	asDict {
-		^this
+	asDict { arg mergeFunc, class;
```

Comment:
Would you mind changing this `arg` to pipes? (Or change pipes to args in `asPairs` above, that seems to be file-consistent)